### DISCUSSION TITLE
How to use hue in iFrame of another website？Are there some instances?

### DISCUSSION THREAD

**Discussion by TaylorVictor** [cloudera/hue#2224] [Questions] [answered]
How to use hue in iFrame of another website？Are there some instances?

**Comment by Harshg999** [comment, marked as answer]
Hi @TaylorVictor! Thank you for your question!

With the release of new SQL Scratchpad web component and public REST APIs out now, you can easily create your own Editor in another website!
Docs - https://docs.gethue.com/developer/components/scratchpad/
Release Blog - https://gethue.com/blog/2021-05-29-create-own-sql-editor-via-webcomponent-and-public-api/

Following the above resources, the SQL Scratchpad can be set up which points to the demo running instance - [demo.gethue.com](https://demo.gethue.com/hue/accounts/login?next=/).

Hope this helps!
